### PR TITLE
By KJH branch test

### PR DESCRIPTION
**Pull request by NotSecretSuperStar**
Commits

### CODE REVIEW

**Comment by StevenHocking** [comment]
Have you modified the key?

**Comment by NotSecretSuperStar** [comment]
Yes, I have modified.

**Comment by StevenHocking** [comment]
OK, I'll check.

File: backend/routes/route.js [hunk 1]
```diff
@@ -27,7 +27,7 @@ router.route("/test").get(function (req, response) {
 router.route("/postdata").post(async (req, response) => {
 
   let data = req.body.data;
-  const API_KEY = "YOUR_GOOGLE_MAP_API_KEY";
+  const API_KEY = "YOUR_GOOGLE_MAP_API_KEY33333";
```
Comment (StevenHocking) [inline backend/routes/route.js:30]: Delete 33333

Comment (StevenHocking) [reply backend/routes/route.js:30]: Checked?

**Review by StevenHocking — COMMENTED** [review]
Comment sended

File: backend/routes/route.js (hunk 1 above)
Comment (StevenHocking) [reply backend/routes/route.js:30]: why

Comment (NotSecretSuperStar) [reply backend/routes/route.js:30]: ???

Comment (StevenHocking) [reply backend/routes/route.js:30]: Leave

**Review by StevenHocking — CHANGES_REQUESTED** [review]
Change the code.

**Review by StevenHocking — APPROVED** [review]

**Pull request merged**